README.md

```diff
@@ -11,6 +11,25 @@ Blog post: https://discuss.ardupilot.org/t/gsoc-2021-custom-firmware-builder/749
 This website uses the Flask library. Flask must be installed before use.  
 Directories: `ardupilot` must be within `base`, which must be in the same directory as `CustomBuild`.
 
+### Directory structure
+The ardupilot directory must be in the same directory as the CustomBuild directory.
+
+`╭─khancyr@pop-os ~/Workspace/ardupilot_custom_build
```

Comment:
This formatting doesn't seem to render properly and is a little confusing anyway, so would probably be best to remove it.

Reply:
done